web/packages/plugins/analysis-platform/src/modules/Asset/AssetCreate.tsx

```diff
@@ -17,6 +17,7 @@ import { AxiosError } from 'axios'
 
 export const AssetCreate = (): JSX.Element => {
   const settings = useContext(ApplicationContext)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore